JDI.Light/JDI.Light.Tests/Utils/FileLogger.cs

```diff
@@ -10,7 +10,7 @@ namespace JDI.Light.Tests.Utils
 {
     public class FileLogger : ILogger
     {
-        public const string LogFilePath = @"C:\temp\jdi.txt";
+        public static string LogFilePath = @"C:\temp\jdi.txt";
```

Comment:
Codacy found an issue: [Change the visibility of 'LogFilePath' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5848342)